src/components/book-list/book-list.html

```diff
@@ -0,0 +1,23 @@
+<div class="master-container list-container">
+  <div class="navbar-container">
+    <div class="navbar">
+      <img class="wolox-logo" src="../../assets/LogoWolox.png" alt="wolox logo">
+      <input class="book-search" ng-model="searchtxt" type="text" placeholder="Buscá por título del libro...">
```

Comment:
Remember to write variables in camelCase (there are some exceptions though), also lets try to be more descriptive with names, everyone knows what "txt" means, but there is no harm in typing "text".
```suggestion
      <input class="book-search" ng-model="searchText" type="text" placeholder="Buscá por título del libro...">
```